docs/sphinx/examples/python/advanced_vqe.py

```diff
@@ -1,22 +1,23 @@
 import cudaq
 from cudaq import spin
 
-from typing import List, Tuple
-
 # We will be optimizing over a custom objective function that takes a vector
 # of parameters as input and returns either the cost as a single float,
 # or a tuple of (cost, gradient_vector) depending on the optimizer used.
 
 # In this example, we will use the spin Hamiltonian and ansatz from `simple_vqe.py`
-# and find the `thetas` that minimize the expectation value of the system.
+# and find the `angles` that minimize the expectation value of the system.
 hamiltonian = 5.907 - 2.1433 * spin.x(0) * spin.x(1) - 2.1433 * spin.y(
     0) * spin.y(1) + .21829 * spin.z(0) - 6.125 * spin.z(1)
 
-kernel, thetas = cudaq.make_kernel(list)
-qubits = kernel.qalloc(2)
-kernel.x(qubits[0])
-kernel.ry(thetas[0], qubits[1])
-kernel.cx(qubits[1], qubits[0])
+
+@cudaq.kernel
```

Comment:
_[At the risk for sounding like a broken record]_ Shouldn't this go into `python_mlir` directory?
I might have misunderstood the intention...

Reply:
Looking at the latest commits. this seems to be resolved.